### DISCUSSION TITLE
/zh-cn/blog/posts/20230514/

### DISCUSSION THREAD

**Discussion by Reverier-Xu** [Reverier-Xu/discussions#121] [Announcements]
https://blog.woooo.tech/posts/20230514/

**Comment by BellssGit** [comment]
感觉i7+2070应该也还不算太差🤔，或许加点内存还能再战，现在DDR4也便宜不少
PS：另外不懂是哪边的问题，Wayland+FF+iBus在评论框打不出字，上面那个n请自动忽略(

**Comment by Reverier-Xu** [comment]
草，半夜牢骚真有人看啊（逃）

再便宜也买不起了，够吃饭付水电费就不错了

**Comment by zeroy0410** [comment]
加油！
（另 我R9+3060的机子一年就寄了，不插电开机即死机，音频模块完全坏掉（绷

**Comment by arttnba3** [comment]
> 感觉生活应该是很光明的，前途看起来是很不错的，可是为什么会这样呢

> 我小时候经常因为遭受到不公平待遇之后被批评不够吃苦不够努力才会这样

看到这句话挺感慨的，尤其是在 C***a[1] 这样的洼地[2]，缺少了对于人文精神的重视，一切对美好的追求似乎都被各种有形与无形的枷锁限制着。自 1915 年开始的新文化运动似乎在某个年代被按下了暂停键，孔儒之魂仍在这片大陆上肆虐着，人与人之间的内斗与内耗似乎仍然是这片土地永恒的精神内核，现代文明似乎离我们越来越远...

本来想再多说点什么，想了想还是算了。一言蔽之，这片土地缺少一次真正完整的思想解放与文艺复兴运动。

> 只会在群里骂这个骂那个，动不动抽象讽刺，除了传递负面情绪之外屁用没有

生活在这片大陆上的年轻人是全世界最独特的一个群体，他们身处在全世界最不公平[3]、贫富差距最大[4]、生存成本最高[5]的一个“发达国家”当中，他们的身边似乎充斥着各种各样的“恶”，他们所遇到的问题似乎都是其他国家的好几倍，但似乎越来越少有人出来呐喊或是做些什么。对于每天发生在他们身边的一切，有的人选择想办法逃离，但更多数人则是沉浸在高位者所构建的奶头乐与幻象当中，无论是主动还是被动，最终都将逐渐成为大酱缸的一部分。

曾经有人想为这一切做些什么，最终都在铁锤下化为了虚无，而对于大部分人而言生命终究是更为重要的事物，于是相较于其他国家动不动就上街游行为自身争取权利的景象，此地的年轻人看起来似乎只有化作网络喷子的本事——因为曾经站出来的人都没了，而似乎也没有人为这些义人呐喊过。

“发达国家”的科技终究处在不低的水平，即使民众选择主动放弃自身应有的政治权利，似乎仍旧能过得很舒适。但铁拳的存在不会随着双目的闭上而消失，总会十分有力地锤向一个又一个的目标，为了克服铁锤存在的恐惧，“逃避”似乎是最简单的事情——于是有的人开始挥刀向着那些呐喊着的更弱者而去，仿佛这样就能让世界看起来美好一些。

但对于直面过“世界真实”的人，他眼前的这片天空已经不再是蓝色的了。

幸运的是，仍然有人愿意为改变这一切贡献一份自己的力量[6]，而那便是黑暗当中唯一一束微小的光。


> 注1：指 Canada，即加拿大，位于北美大陆的一个国家
>
> 注2：洼，音同 "wonderful" ，即“美好富足的土地”
> 
> 注3：平行空间历史 - 自[本案](https://zh.wikipedia.org/wiki/%E5%A7%9C%E5%A6%8D%E3%80%81%E6%A0%BE%E5%80%A9%E3%80%81%E5%BC%A0%E5%A4%A9%E7%8F%A0%E8%AF%89%E6%95%99%E8%82%B2%E9%83%A8%E6%A1%88)之后，美国各州的 College Entrance Examination（CEE）考试开始实行独立命题
> 
> 注4：平行空间历史 - [日本贫富差距](https://zh.wikipedia.org/wiki/%E4%B8%AD%E5%8D%8E%E4%BA%BA%E6%B0%91%E5%85%B1%E5%92%8C%E5%9B%BD%E8%B4%AB%E5%AF%8C%E5%B7%AE%E8%B7%9D)
>
> 注5：平行空间历史 - [乌克兰土地财政](https://zh.wikipedia.org/wiki/%E5%9C%9F%E5%9C%B0%E8%B4%A2%E6%94%BF) 与 [韩国房价问题](https://zh.wikipedia.org/wiki/%E4%B8%AD%E5%9B%BD%E4%BD%8F%E6%88%BF%E9%97%AE%E9%A2%98)
>
> 注6：~~数据删除~~

> 补充说明一下，很多问题其实不仅仅是小区物业的问题，更是全体小区居民的问题，整个小区本就生活在一个令人窒息的文化环境当中，从此当中选出的物业自然不用多想，而这样的物业则又将小区带向更糟的境地，形成一个恶性循环。而这样的循环实际上已经经历了数千年。正如黑格尔在《法哲学原理》一书中说=所言——"西韩的历史从本质上看是没有历史的，它只是君主覆灭的一再重复而已。任何进步都不可能从中产生。"
>
> 其他小区或许也或多或少存在不同的问题，但至少笔者认为一个去年考10分今年考9分的小区似乎没有多少资格评论那些80甚至90分的小区（笑）。

**Comment by orangecheers-x** [comment]
比起改变整个环境, 对于其中最弱小的, 只想做好自己的事的底层人来说, 逃离似乎更简单一些.

虽然不知道哪里是天堂, 到底有没有天堂, 但是我知道这地方一定是地狱.

如果问题出在整个体系上, 底层的人想要通过恪守职责来改变整个环境, 这件事放眼世界历史可能都找不到几个. 即使是想要改良, 最初也要靠有权的人才能真正的发起. 即使是小到西电, 我觉得也是这样的.

如果真的想要改变而不是逃离, 那么不在沉默中爆发, 就在沉默中灭亡.

无论怎样, 只要想要的和在做的方向是一致的, 我觉得你的执行力会让你成功的.

**Comment by BenderBlog** [comment]
“这台老伙计，19年的i7+2070，大概算很高配的东西，花了近1w8大洋”
tfl 我这 20 年的 Thinkpad T14 用到现在也很流畅，是不是 Arch 大法加成了......

**Comment by Reverier-Xu** [comment]
> “这台老伙计，19年的i7+2070，大概算很高配的东西，花了近1w8大洋” tfl 我这 20 年的 Thinkpad T14 用到现在也很流畅，是不是 Arch 大法加成了......

买的时候属于是倾家荡产了……

其实这配置放到现在也还彳亍，不过这四年下来几乎没关过机，天天被C++和Rust编译摧残，身上部件没几个好的，Intel那qlc ssd都快被写报废了

**Comment by ZeroAurora** [comment]
某天深夜 Rx 和我吐槽这事来着，现在回来看居然写成了博客。

能够遇到一个同样理想主义者的朋友确实是三生有幸。

---

看了眼评论区有点反胃，打了半天字又删掉。突然又想到了 core-js，有空再给他打点钱吧。

---

我知道你想问什么，我要走的是一条荆棘之路，我真的下定决心了吗？......我不知道未来的我会不会后悔，但是，至少现在，我会坚定地走下去。

Ref=>ak_char:淬羽赫默/语音记录/精英化晋升2

**Comment by orangecheers-x** [comment]
> @ZeroAurora
> 某天深夜 Rx 和我吐槽这事来着，现在回来看居然写成了博客。
> 
> 能够遇到一个同样理想主义者的朋友确实是三生有幸。
> 
> ---
> 
> 看了眼评论区有点反胃，打了半天字又删掉。突然又想到了 core-js，有空再给他打点钱吧。
> 
> ---
> 
> 我知道你想问什么，我要走的是一条荆棘之路，我真的下定决心了吗？......我不知道未来的我会不会后悔，但是，至少现在，我会坚定地走下去。
> 
> Ref=>ak_char:淬羽赫默/语音记录/精英化晋升2

无论你指的是谁，都不妨把话说得更明白一些。这些言论不像是想与人友善交流的样子，只会让我感到冒犯。

**Comment by arttnba3** [comment]
> @ZeroAurora
> 
> 看了眼评论区有点反胃，打了半天字又删掉

我大概能理解你想说的是什么，如果我想要短时间内说服你我自己也觉得不可能（笑），不过我觉得你应该思考一下 **“问题究竟是出在【把现状说出来的人】，还是【造成了现状的人】”**

以及很多事情不是你闭上眼睛不去看他就会消失、就会变好的 ：）

**Comment by ZeroAurora** [comment]
我是知道啊，谁不知道呢？这都不知道属于是山顶洞人。
那些“造成现状的人”，我也很想上去给他们几巴掌。

但所谓的 C***a，洼地就很明白？这里好歹是 GitHub Issues/Discussions，大可以再放开点。
我把话咽下去，不去交流避免骂战，也算一种友善交流。关于冒犯，我很抱歉，我本不该写出来。

**Comment by orangecheers-x** [comment]
> 我是知道啊，谁不知道呢？这都不知道属于是山顶洞人。
> 那些“造成现状的人”，我也很想上去给他们几巴掌。
> 
> 但所谓的 C***a，洼地就很明白？这里好歹是 GitHub Issues/Discussions，大可以再放开点。
> 我把话咽下去，不去交流避免骂战，也算一种友善交流。关于冒犯，我很抱歉，我本不该写出来。

好的

**Comment by arttnba3** [comment]
> @ZeroAurora
> 
> 我是知道啊，谁不知道呢？这都不知道属于是山顶洞人。
>
> 这里好歹是 GitHub Issues/Discussions，大可以再放开点。

这两句话结合起来， **我想说【你不知道】** ，GitHub 是什么法外之地吗？ **你和我都是中国人，不出意外的话我们这辈子永永远远都会是中国人，不要表现得像一个刚到中国大陆的外宾一样** （笑）

> 但所谓的 C***a，洼地就很明白？

关于【洼地】这个词，首先我要说的是， **我并不是一个种族歧视者** ，我不认为关于这个词所涉及到的东西是所谓【中国人的劣根性】（笑），所以不需要看到这样的词就反应特别大。简而言之，我实际上想说的是（提前警告， **下面的这一段话和所谓的“古墓派公知”相类似** ，如果你非常看不爽这一类言论，请直接跳过；如果你愿意心平气和地深入交流，请继续阅读）：

- “在【中华文化】当中有着很多的糟粕，时至今日不仅尚未能完全去除，其中有着相当多一部分更是变本加厉”。

同世界上与我们处在同一经济、工业发展水平的国家相比，【精神文明】这一层面我们是相对低了很多的——因此【洼地】这个词，我认为并无不妥——当然，这并不代表西方文明就是完美无瑕的（需要注意，在交流的时候，我们应当避免从一个极端走向另一个极端， **我并不是一个崇洋媚外的人，我只看对错，不看立场** ），他们自身也存在很多各种各样的问题，但二者有问题的程度终究是不同的，西方文明在 LGBTQ、人权平等、言论自由等各个层面终究是做得比我们要好很多的。

> 需要说明的是，这里 **暂时不讨论 politics 的因素** ，仅讨论文明本身

**我并不认为中国人做不到这一点，但如今社会的氛围确乎并没有向着更加开放的环境发展，反而似乎是在逐步紧缩** ，80年代末我们还能上街游行反贪腐[1]，21世纪初人们还能公开辩论郭敬明和韩寒谁是攻谁是受[2]，那英还能在微博上说“妈的最烦装逼的人”，创作者们还能自由地表达自己的想法，两岸三地网友也能友善地进行交流......但到了现在，LGBTQ 人群依然不能生活在阳光之下[3]，一提到同性恋最低限度的歧视也是“我不反对但请你们不要出现在大家面前”；社交平台上的每一个字都要受到大家的审查，说错一个字就是辱华就得退场；网络上大家好像没有对错只有立场，极端男女拳分裂骂战不断，大家一提到俄罗斯入侵乌克兰就叫好，一提到黑人就喊尼哥；现实生活中我们仍旧没有更多的喘息，中考、高考、考研，稍微 Gap 一下就被社会丢下[4]，整个人生仿佛永远在和什么东西赛跑着......

### 明明 1915 年就开始新文化运动了，怎么我们的社会现在反倒变成这个样子了？

> 注0：我不是 LGBTQ 人群，也不是女权人士，但今天我不为他们说话，明天也没有人会为我说话 ：）
> 
> ~~注1：数据删除~~
> 
> 注2：华东理工大学辩论赛《韩寒和郭敬明该不该在一起？》[B站链接](https://www.bilibili.com/video/BV1cE411x7B8/)
> 
> 注3：其实中国是亚洲最早开放同性婚姻的国家，只不过只在台湾省开放（笑）
> 
> 注4：对于绝大部分企业而言非应届生只能走社招入职，无论你此前是否有工作经验

**Comment by ZeroAurora** [comment]
> @arttnba3

感谢解读，现在明晰了许多，并且我也表示支持。
狡辩一下，我认为“洼地”“不明白”的原因仅在于我不知道它的具体意义，抱歉。
（Update：额外狡辩一句，既然都“疏而不漏”了，那其实有没有避讳，最后的结果差得都不大。等个 Rx 过来删评。）

**Comment by xeonds** [comment]
理想主义者在现实中总会多少感受到痛苦，坚持着走下去就好，或许。

另，CTF平台原来是您写的，膜一个。前端很惊艳，后端思路也很有意思。

再另，原来当时活动的访客登记系统也是您写的？~~就说url这么眼熟~~再膜一个。

>震撼于您的代码产出能力
